requirements.txt

```diff
@@ -1,31 +1,51 @@
+alabaster==0.7.12
 alembic==1.4.2
 aniso8601==8.0.0
+Babel==2.8.0
 bcrypt==3.1.7
 certifi==2020.6.20
 cffi==1.14.0
 chardet==3.0.4
 click==7.1.2
+colorama==0.4.4
+coverage==5.3
```

Comment:
this should also go in requirements-dev.txt I think.

Reply:
I will fix the requirements files and try to have them correctly separated